crates/l1tx/Cargo.toml

```diff
@@ -16,6 +16,7 @@ thiserror.workspace = true
 tracing.workspace = true
 
 [dev-dependencies]
+hex.workspace = true
```

Comment:
But you don't need that. You have `bitcoin` in your dependencies and you can use `use bitcoin::hex_conservative as hex`

See the docs here: https://docs.rs/hex-conservative/0.2.0/hex_conservative/index.html